### PR TITLE
Update genie.libs.parser and genie.libs.robot

### PR DESCRIPTION
**Pull request by kecorbin**
@siming85 Take a peek, this works for my needs but  I felt like the version changes required here were starting a pattern that may get out of hand quickly with all the various libs.  Perhaps we could freeze a requirements.txt right beside the Dockerfile?

### CODE REVIEW

File: 4.1/robot/Dockerfile [hunk 1]
```diff
@@ -21,8 +21,8 @@ RUN pip3 install --upgrade --no-cache-dir setuptools pip virtualenv
 # version definitions
 ARG PYATS_VERSION=4.1.0
 ARG PYATS_ROBOT_VERSION=4.1.1
-ARG GENIE_VERSION=3.0.0
-ARG GENIE_ROBOT_VERSION=3.0.7
```
Comment (kecorbin) [inline 4.1/robot/Dockerfile]: This was a bit confusing to me (at first) as I was not accounting for all the other genie packages.

File: 4.1/robot/Dockerfile [hunk 2]
```diff
@@ -21,11 +21,14 @@ RUN pip3 install --upgrade --no-cache-dir setuptools pip virtualenv
 # version definitions
 ARG PYATS_VERSION=4.1.0
 ARG PYATS_ROBOT_VERSION=4.1.1
-ARG GENIE_VERSION=3.0.7
+ARG GENIE_VERSION=3.0.0
+ARG GENIE_PARSER_VERSION=3.0.7
 ARG GENIE_ROBOT_VERSION=3.0.8
```
Comment (kecorbin) [inline 4.1/robot/Dockerfile]: And this seems like it could get out of control in the future...

Comment (siming85) [reply 4.1/robot/Dockerfile]: yeah because installing Genie auto installs everything else. , except robot, which is separate.

**Comment by siming85** [comment]
pushed out new images to dockerhub for genie/pyats with and without robot

**Pull request closed without merging**